harness/decimalToHexString.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2017 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+function decimalToHexString(n) {
+  var hex = "0123456789ABCDEF";
+  n >>>= 0;
```

Comment:
It's basically just performing `n = ToUin32(n)`, but it actually doesn't really matter here. We could also use `n |= 0` to cast it into an signed int32, or even just omit the (u)int32 cast at all, because we _know_ all call sites will supply sensible values.